pandas/io/sas/sas_xport.py

```diff
@@ -279,6 +279,8 @@ def _read_header(self):
         # read file header
         line1 = self._get_row()
         if line1 != _correct_line1:
+            if "**COMPRESSED**" in _correct_line1:
```

Comment:
Happy to. Is it ok to use [assertRaises](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises)? I ask because the other tests don't use the `unittests` module, and I couldn't find an example of testing for a specific exception.

Reply:
```
with pytest.raises(FooError, match=regexp_for_exception_message):
     func()
```

Reply:
Done. Adding the unit test showed that I had a bug, which I fixed, and the tests are passing now. (As an aside, I missed that the instructions to add unit tests for changes and to use pytest are right there in the [Contributing to pandas](https://pandas.pydata.org/pandas-docs/version/1.2.4/development/contributing.html#test-driven-development-code-writing) doc. I really appreciate the patience and guidance 🙂)